### ISSUE TITLE
cloud-server should remove the job folder when `paddlecloud kill <job name>`

### ISSUE THREAD

**Issue by Yancey1989** [PaddlePaddle/cloud#292]

**Comment by typhoonzero** [comment]
No, we shouldn't. Job fold may contain user writed log files, outputs, etc. Let user delete them is safer.

**Comment by Yancey1989** [comment]
But there maybe a lot of useless files, such as core.xxxx, maybe we can delete them with a cron job?

**Comment by typhoonzero** [comment]
No, I think we should never delete user's stuff. Maybe add some document about when to delete files.
